Makefile

```diff
@@ -120,13 +128,14 @@ test-e2e: $(GINKGO) ## Run the end-to-end tests
 .PHONY: linters
 linters: lint generate-check fmt-check
 
-tools/bin/golangci-lint: hack/tools/go.mod
-	cd hack/tools; go build -o $(abspath $@) github.com/golangci/golangci-lint/cmd/golangci-lint
+$(GOLANGCI_LINT):
```

Comment:
Can you add phony target for $(GOLANGCI_LINT_BIN) so it is easier to call as is? Now you need to call `make $(pwd)/hack/tools/bin/golangci-lint` to make it.

Reply:
I can also add it in follow-up, if not feeling like it.

Reply:
I will add it here in a while

Reply:
Done

Reply:
Thanks. 

/lgtm